rsconf/component/rsiviz.py

```diff
@@ -45,6 +45,9 @@ def internal_build_compile(self):
             # is there a way to just use it and not specify cmd?
             docker_exec=f"bash {db.user_home_path(z._run_u)}/.radia-run/start",
         )
+        self.buildt.get_component("network").add_public_tcp_ports(
```

Comment:
add_public_tcp_ports should convert to str, not here.

Reply:
38327fc